code/game/turfs/turf.dm

```diff
@@ -79,12 +79,14 @@
 		spawn( 0 )
 			src.Entered(AM)
 			return
-	turfs |= src
```

Comment:
Also @Fox-McCloud there's no way to correctly add z levels right now anyways, so I doubt it's an issue ATM.